crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -436,6 +436,7 @@ impl Schedule {
         &mut self.graph
     }
 
+    #[cfg(feature = "bevy_debug_stepping")]
```

Comment:
why is this feature flagged now?

Reply:
it's because this function is only used in `stepping.rs` and my code wasn't compiling without it. Should I remove it?